signup-front/src/components/molecules/ContactTable.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+const contacts = [
+  { name: 'AidantConnect', address: 'contact@aidantsconnect.beta.gouv.fr' },
```

Comment:
the name attributes is redundant with TARGET_API_LABELS constant defined in signup-front/src/lib/api.js